src/mdio/creators/mdio.py

```diff
@@ -0,0 +1,80 @@
+"""Creating MDIO v1 datasets."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from segy.standards import get_segy_standard
+
+from mdio.api.io import _normalize_path
+from mdio.api.io import to_mdio
+from mdio.builder.template_registry import TemplateRegistry
+from mdio.builder.xarray_builder import to_xarray_dataset
+from mdio.converters.segy import get_horizontal_coordinate_unit
+from mdio.converters.segy import populate_dim_coordinates
+from mdio.converters.type_converter import to_structured_type
+from mdio.core.grid import Grid
+
+if TYPE_CHECKING:
+    from pathlib import Path
+
+    from upath import UPath
+    from xarray import Dataset as xr_Dataset
+
+    from mdio.builder.schemas import Dataset
+    from mdio.core.dimension import Dimension
+
+
+def create_empty_mdio(  # noqa PLR0913
```

Comment:
```suggestion
def create_empty(  # noqa PLR0913
```

Suffixing everything as `_mdio` feels redundant to me